Riot/Modules/LaunchLoading/LaunchLoadingViewController.swift

```diff
@@ -0,0 +1,38 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import Reusable
+
+class LaunchLoadingViewController: UIViewController, Reusable {
```

Comment:
I always liked the "SplashScreen" name 😁

Reply:
Too many options! I followed `LaunchLoadingView` as a convention here, but there are designs to update the loading view at some stage, so maybe at that point we can switch it 💦